Automattic/wp-calypso
# Config: Extract languages to shared config and remove obsolete muse_supported_themes

We repeat across all config files so many values. We could easily mitigate that issue by using already present `_shared.json` config file introduced by @mjangda in #876. This PR tries to slim down config files by extracting `muse_supported_themes` and `languages` fields. We probably can open the follow up PR with more changes.

### Testing
1. Run `npm test` and make sure everything passes.
2. Run `http://calypso.localhost:3000/me/account` and make sure you still can change language for your interface :)
3. Go to your site's setting page and check if you can change language of your site.
